addons/event_sale/models/event.py

```diff
@@ -13,7 +13,6 @@ class event_event(models.Model):
         'event.event.ticket', 'event_id', string='Event Ticket',
         default=lambda rec: rec._default_tickets(), copy=True)
 
-    badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
```

Comment:
Shouldn't all the badge fields grouped in the same module ? Or is there a special reason for these 2 fields to be in `event_sale` ?
@tde-banana-odoo your opinion ?